client/routes.js

```diff
@@ -1,25 +1,18 @@
+// react lib imports
 import React from 'react';
-import { Route, IndexRoute } from 'react-router';
+import { Route, Switch } from 'react-router-dom';
 
 // components
-import App from './components/App';
```

Comment:
Why are we removing all of these?

Reply:
It won't be needed as a dashboard page will be created that will serve as the application entrance page and would also render other components within it.

Reply:
yea, eventually. but we should add back all of those to avoid breaking changes. When we get to eventually work on those, that will be removed

Reply:
yea, eventually. but we should add back all of those to avoid breaking changes as long as it does not affect your own implementation. When we get to eventually work on those, that will be removed